bigtable-client-core-parent/bigtable-hbase/src/main/java/com/google/cloud/bigtable/hbase/wrappers/veneer/DataClientVeneerApi.java

```diff
@@ -134,6 +140,17 @@ public Result apply(Row row) {
         MoreExecutors.directExecutor());
   }
 
+  @Override
+  public ResultScanner readRows(Query.QueryPaginator paginator) {
+    return new PaginatedRowResultScanner(
+        paginator,
+        (p) -> {
+          return delegate
+              .readRowsCallable(RESULT_ADAPTER)
+              .call(p.getNextQuery(), createScanCallContext());
+        });
```

Comment:
nit: you dont need the curly brackets or return
```
p -> delegate
              .readRowsCallable(RESULT_ADAPTER)
              .call(p.getNextQuery(), createScanCallContext());
```

Reply:
Done